### DISCUSSION TITLE
Willkommen zur ESPLesekopf

### DISCUSSION THREAD

**Discussion by esplesekopf** [esplesekopf/esplesekopf#4] [General]
## 👋 Hallo
  Hier gibt es die Möglichkeit über den Lesekopf zu diskutieren und Ideen auszutauschen
  * bleibt nett miteinander
  * Konstruktiv 
  * Erwartet nicht das ich auf alles antworte oder auf alles Reagiere. Für Besitzer des Lesekopfs gibt es andere Kommunikationswege!
  * Ich mache das ganze in meiner Freizeit und mit Familie ist nicht soo viel davon da.

Na dann mal los!

**Comment by NemoN** [comment]
Frage: Kann man eine fertig bestückten ESPLesekopf Platine irgendwo kaufen?

**Reply by esplesekopf** [reply]
Natürlich kann man den Lesekopf kaufen 😁. Schreib mir eine Mail an esplesekopf@gmail.com

**Reply by NemoN** [reply]
Sorry, im READ.me steht es ja.

**Reply by DiKaY1969** [reply]
Servus,
ich würde gerne auch einen weiteren Lesekopf mit Gehäuse kaufen.
Der erste von dir läuft seit 3-Jahren tadellos und jetzt benötige ich noch einen zweiten wegen einer Zählerkaskade.
Ich versucht dich schon seit 2-Wochen per E-Mail zu kontaktieren was aber bisher nicht geklappt hat. Vielleicht bist du ja in Urlaub oder im Stress. Es wäre nett wenn du mich bitte kontaktieren könntest.

Vielen Dank im voraus.

Grüße
Dierk
